lib/administrate/field/polymorphic.rb

```diff
@@ -30,7 +30,7 @@ def associated_dashboard(klass = data.class)
       end
 
       def classes
-        options.fetch(:classes) || []
+        options.fetch(:classes, [])
```

Comment:
`.fetch` raises a KeyError when the key is missing, it doesn't return `false`, so the second part of the conditional would have never ran in the previous code:

```
2.4.0 :001 > hash = {}
 => {} 
2.4.0 :002 > res = hash.fetch(:key1) || 'Something'
KeyError: key not found: :key1
	from (irb):2:in `fetch'
	from (irb):2
	from /Users/fana/.rvm/rubies/ruby-2.4.0/bin/irb:11:in `<main>'
2.4.0 :003 > res
 => nil 
```

I'm wondering if users have been rescuing `KeyError` as a result and this will be a breaking change for them?

Reply:
I think if someone were rescuing then it would be to return a default. In that case, the default is likely also an empty Array as it is consistent with other keys from `options`. Where do you think a user may have added the rescue? If they were overriding this method then this change would not affect them. 🤔 What do you think?